manifest.json

```diff
@@ -12,48 +12,61 @@
     "content_scripts": [
         {
             "matches": [
-                "*://*.steamcommunity.com/market/listings/730/*",
-                "*://*.steamcommunity.com/id/*/inventory*",
-                "*://*.steamcommunity.com/profiles/*/inventory*",
-                "*://*.steamcommunity.com/tradeoffer/*"
+                "*://*.steamcommunity.com/market/listings/730/*"
             ],
             "js": [
-                "lib/filtrex.js",
-                "lib/utils.js",
-                "lib/filters.js",
-                "lib/queue.js",
-                "lib/bridge.js",
-                "lib/trade_offer.js",
-                "lib/stall.js",
-                "lib/inventory_fallback.js",
-                "lib/banner.js",
-                "float.js"
-            ],
-            "css": ["float.css"]
+                "src/lib/page_scripts/market_listing.js"
+            ],
+            "css": ["src/global.css"]
         }, {
             "matches": [
                 "*://*.steamcommunity.com/id/*/tradehistory*",
                 "*://*.steamcommunity.com/profiles/*/tradehistory*"
             ],
             "js": [
-                "lib/utils.js",
-                "lib/time_fetcher.js"
+                "src/lib/page_scripts/time_fetcher.js"
+            ],
+            "css": ["src/global.css"]
+        }, {
+            "matches": [
+                "*://*.steamcommunity.com/id/*/inventory*",
+                "*://*.steamcommunity.com/profiles/*/inventory*"
+            ],
+            "js": [
+                "src/lib/page_scripts/inventory.js"
             ],
-            "css": ["float.css"]
+            "css": ["src/global.css"]
+        }, {
+            "matches": [
+                "*://*.steamcommunity.com/tradeoffer/*"
+            ],
+            "js": [
+                "src/lib/page_scripts/trade_offer.js"
+            ],
+            "css": ["src/global.css"]
         }
     ],
     "background": {
-        "scripts": ["background.js"]
+        "service_worker": "src/background.js",
+        "type": "module"
     },
     "permissions": [
+        "storage",
+        "scripting"
```

Comment:
Of note, this is a newly requested permission and needed since you aren't allowed to inject strings as JS into the page anymore.

This however, should **not** trigger a warning for users and auto-disable the extension:
![image](https://user-images.githubusercontent.com/11811793/193465596-1eb44266-d536-43bc-bc4e-07af13a83dc0.png)

https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#executing-arbitrary-strings